platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/isolated/IsolatedProjectsAccessFromGroovyDslIntegrationTest.groovy

```diff
@@ -866,6 +878,7 @@ class IsolatedProjectsAccessFromGroovyDslIntegrationTest extends AbstractIsolate
         }
     }
 
+    @Ignore("configuration is parallel with IP, so unconfigurance is unpredictable")
```

Comment:
💡 Can we avoid ignoring the test if we replace `:b` with `:a:sub`?

Reply:
Maybe, but I intentionally didn't alter the tests